README.adoc

```diff
@@ -1 +1,20 @@
-# tensorflow
+# Spring Cloud Stream TensorFlow Processor
+
+A processor that evaluates a machine learning models build with link:https://www.tensorflow.org/[TensorFlow] and stored
```

Comment:
Fine with me either way.
Only inconvenience that we will get notifications for all those commits 😄